schemas/initial_stars_detector_standardized.yaml

```diff
@@ -0,0 +1,196 @@
+# This file defines the mapping between the columns in a single visit+detector
+# initial_stars table and their respective DPDD-style column names, as used by
+# `lsst.pipe.tasks.postprocess.TransformSourceTableTask`.
+# See the DPDD for more information about the output: https://lse-163.lsst.io
+funcs:
+    sourceId:
+        functor: Column
+        args: id
+    coord_ra:
+        # reference position required by db. Not in DPDD
+        functor: CoordColumn
+        args: coord_ra
+    coord_dec:
+        # Reference position required by db. Not in DPDD
+        functor: CoordColumn
+        args: coord_dec
+    # objectId: not avaliable
+    # ssObjectId: not avaliable
+    parentSourceId:
+        functor: Column
+        args: parent
+    # htmId20: not avaliable
+    x:
+        functor: Column
+        args: slot_Centroid_x
+    y:
+        functor: Column
+        args: slot_Centroid_y
+    xErr:
+        functor: Column
+        args: slot_Centroid_xErr
+    yErr:
+        functor: Column
+        args: slot_Centroid_yErr
+    # x_y_Cov: not available
+    ra:
+        functor: RAColumn
+    dec:
+        functor: DecColumn
+
+   # RFC-924: Temporarily keep a duplicate "decl" entry for backwards
+   # compatibility. To be removed after September 2023.
```

Comment:
Should the post-September 2023 removal be a part of this ticket?

Reply:
It definitely shouldn't be part of this ticket; I think it might be disruptive (hence the RFC).  And while September 2023 has long passed, I think that was a "no earlier than" date, not a deadline.